src/Elastic.Markdown/Slices/HtmlWriter.cs

```diff
@@ -106,6 +106,8 @@ private async Task<string> RenderLayout(MarkdownFile markdown, MarkdownDocument
 
 		var siteName = DocumentationSet.Tree.Index?.Title ?? "Elastic Documentation";
 
+		var previousVersionUrl = DocumentationSet.Build.HistoryMapper.MapPreviousUrl(markdown.YamlFrontMatter?.MappedPages?.FirstOrDefault());
```

Comment:
```suggestion
		var legacyUrl = DocumentationSet.Build.HistoryMapper.MapPreviousUrl(markdown.YamlFrontMatter?.MappedPages?.FirstOrDefault());
```

Rename all `PreviousVersionUrl` occasions with `LegacyUrl` to make it clear its has not something to do with pagination of versioning of content in the new v3 system.